rfcs/0005-pablo-distribution-assets/0005-pablo-distribution.adoc

```diff
@@ -505,19 +505,7 @@ Following algorithm should be part of the block hook in the pallet.
 
 [plantuml,images/staking-rewards-reward-accumulation-hook,png]
 ----
-start
-->last_update_timestamp;
-:elapsed_time = current_time - last_update_timestamp;
-while (assetId = stake_pools.has_next) is (yes)
-    :rewards_pool = readStorage(assetId);
-    while (reward = rewards_pool.rewards.has_next) is (yes)
-        :assert max_reward not exceeded;
-        :reward.total_rewards += elapsed_time * reward.reward_rate;
-    endwhile (no)
-    :save(rewards_pool);
-endwhile (no)
-:last_update_timestamp = current_time;
-stop
+c
```

Comment:
Mistake?

Reply:
Yep, not sure how that happened. Fixed: https://github.com/ComposableFi/composable/pull/1359/commits/323f56203262bea61fa2fd252ca961553b535af3